examples/TypeScriptMessaging/App.tsx

```diff
@@ -228,7 +228,6 @@ const App = () => {
         theme={{
           colors: {
             ...(colorScheme === 'dark' ? DarkTheme : DefaultTheme).colors,
-            background: theme.colors?.white_snow || '#FCFCFC',
```

Comment:
Not needed in TSMessagingApp. Since removing it will give us a fair idea about the component's default theme.